.github/workflows/artifacts.yml

```
@@ -0,0 +1,29 @@
+# Builds and commits the artifacts whenever a change is pushed to main.
+name: artifact compiler
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install latest stable
+        uses: ATiltedTree/setup-rust@v1
+        with:
+          rust-version: stable
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build Artifacts
+        run: |
+          ./devtools/optimize.sh
+      - name: Commit artifacts
+        run: |
+          git config --global user.name 'CyberHoward'
+          git config --global user.email 'CyberHoward@users.noreply.github.com'
```

```suggestion
          git config --global user.name 'DA0-DA0'
          git config --global user.email 'DA0-DA0@users.noreply.github.com'
```

Let's use these? Or maybe just `Polytone` for `user.name`?

---

Ah, but we don't have the GitHub org or user name for Polytone so let's just go with `DA0-DA0`.